.github/RELEASE_TEMPLATE.md

```diff
@@ -1,150 +1,114 @@
-  ```
+## Known Issues
 
-3. (optional) Install the bash/zsh auto-completion script:
-
-  ```bash
-  $ mv $HOME/.local/bin/cardano-wallet.sh > /etc/bash_completion.d/cardano-wallet.sh
-  $ source /etc/bash_completion.d/cardano-wallet.sh
-  ```
-
-4. Start `cardano-wallet --help` and see available parameters.
+<!-- Bugs known at the moment of the release, or discovered after and not fixed -->
 
-#### TroubleShooting
-
-<details>
-  <summary>cardano-wallet: error while loading shared libraries: <code>libcrypto.so.1.0.0</code>: cannot open shared object file: No such file or directory</summary>
-  
-`libcrypto 1.0.0` is no longer shipped with RHEL for a while (since ver. 6) and
-SuSE. On these distributions, it is impossible to install it using native
-package manager (e.g. `yum`).
+## Changelog
 
-One possible work-around is to create an artificial symbolic link from a newer version of `libcrypto`:
 
-```bash
-sudo ln -s /usr/lib64/libcrypto.so.1.1.1 /usr/lib64/libcrypto.so.1.0.0
-```
+<!-- A CHANGELOG, organized in milestones. Ideally, we put it within
+some <details></details> elements to avoid cluttering the release notes -->
 
-Alternatively, one may also try using the docker image instead.
-</details>
+{{CHANGELOG}}
 
-## Mac OS 64-bit 
 
-1. Install [jormungandr@X.Y.Z](https://github.com/input-output-hk/jormungandr/releases/tag/vX.Y.Z) from the official repository.
+## Weekly Reports
 
-2. Download `cardano-wallet-jormungandr@vYYYY-MM-DD.macos64.tar.gz` and uncompress it in a directory that is on your `$PATH`, e.g. `/usr/local/bin`.
+- [Week 12 - 2020-03-20](https://github.com/input-output-hk/cardano-wallet/tree/weekly-reports/2020-03-20)
 
-*Note:* Make sure all `*.dylib` files are in the same directory as `cardano-wallet` binary.
+## Documentation
 
-## Windows 64-bit
+<!-- A snapshot of the documentation at the time of releasing. -->
 
-1. Install [jormungandr@X.Y.Z](https://github.com/input-output-hk/jormungandr/releases/tag/vX.Y.Z) from the official repository.
+| Link                                                                                                                                        | Audience                                                   |
+| ---                                                                                                                                         | ---                                                        |
+| [API Documentation](https://input-output-hk.github.io/cardano-wallet/api/{{GIT_TAG}})                                                       | Users of the Cardano Wallet API                            |
+| CLI Manual: [jormungandr](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface/{{JORM_CLI_WIKI_COMMIT}}) / [byron](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface-(cardano-wallet-byron)/{{BYRON_CLI_WIKI_COMMIT}}) | Users of the Cardano Wallet API                            |
```

Comment:
This one should also be updated to be more consistent with the naming, it's either ITN vs Byron or, jormungandr vs cardano-node. 

Also, it feels kinda weird to speak about the `jormungandr cli manual` for the cardano-wallet command-line. What about something like `w/ jormungandr` and `w/ cardano-node` and then refer to them as `cardano-wallet w/ jormungandr cli manual` & `cardano-wallet w/ cardano-node cli manual` ?

Reply:
I'd go with ` ITN / Byron` then because it fits on a single line.